File-Formats/XlsIO/Working-with-Data.md

```diff
@@ -1513,7 +1513,93 @@ namespace ImportFromNestedCollection
 }
 {% endhighlight %}
 
-{% highlight c# tabtitle="C# [Windows-specific]" %}
+A complete working example to import data to Excel worksheet with layout option in C# is present on [this GitHub page](https://github.com/SyncfusionExamples/XlsIO-Examples/tree/master/Import%20and%20Export%20Data/Layout%20Options).
+
+The following screenshot represents the output document with Default layout option.
+
+<img src="working-with-data_images/file-formats-xlsio-default-layout-option.png" alt="Output document with Default layout option" width="100%" Height="Auto"/>
+
+##### Merge layout option
+
```

Comment:
Why did you include this new section? Is this reviewed content?

Reply:
The changes are not reflecting in my md file, I did not make them